frontend/javascripts/test/sagas/saga_integration.mock.ts

```diff
@@ -1,32 +1,30 @@
-import mockRequire from "mock-require";
-import * as antd from "antd";
-import _ from "lodash";
-import "test/mocks/lz4";
+import "antd";
+import { vi } from "vitest";
 
-const REQUEST_ID = "dummyRequestId";
-const UidMock = {
-  getUid: () => REQUEST_ID,
-};
-
-mockRequire("libs/uid_generator", UidMock);
-
-mockRequire("antd", {
-  ...antd,
-  Dropdown: {},
-  message: {
-    hide: () => {},
-    // These return a "hide function"
-    show: () => () => {},
-    loading: () => () => {},
-    success: () => () => {},
-  },
-});
-
-mockRequire("libs/toast", {
-  error: _.noop,
-  warning: _.noop,
-  close: _.noop,
-  success: _.noop,
+vi.mock("antd", () => {
+  return {
+    theme: {
+      getDesignToken: () => ({ colorPrimary: "white" }),
+      defaultAlgorithm: {},
+    },
+    Dropdown: {},
+    message: {
+      hide: vi.fn(),
+      // These return a "hide function"
+      show: vi.fn(),
+      loading: vi.fn(),
+      success: vi.fn(),
```

Comment:
apparently, no test hits this case currently, as the ci passes, but I remember that we had some scenarios in the past were these definitions were important (so that app code can do: hideFn = toast.show(); hideFn()).
therefore, I'd change this to:

```suggestion
      show: vi.fn(() => () => {}),
      loading: vi.fn(() => () => {}),
      success: vi.fn(() => () => {}),
```

or maybe more readable:

```suggestion
      show: vi.fn(() => _.noop),
      loading: vi.fn(() => _.noop),
      success: vi.fn(() => _.noop),
```